### DISCUSSION TITLE
Install & run without internet connection?

### DISCUSSION THREAD

**Discussion by jerverme** [cloudnative-pg/cloudnative-pg#3398] [Q&A] [answered]
Would it be possible to install and run the operator without a connection to the internet, e.g. in highly secure environments?

(Of course it wouldn't be trivial -- during setup you'd need to get the postgres docker image from a private repo.)

**Comment by gbartolini** [comment, marked as answer]
Yes. However, this is not a CloudNativePG domain problem. It is a Kubernetes domain problem, as I imagine CloudNativePG won't be the only workload running in an air-gapped system. Thanks.

**Reply by jerverme** [reply]
Thank you!
